files/common/etc/profile.d/set-umask-for-all-users.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+umask 022
```

Comment:
022 is the current default umask. I think CIS checks are looking for 027. Can you verify that?

Reply:
Actually, the CIS check report says,

```
Remediation: 
Edit file '/etc/bash.bashrc' to configure 'UMASK' setting according to the business needs and organization's security policies.

Edit the 'etc/profile and /etc/profile.d/*.sh' file to configure 'umask' setting according to the business needs and organization's security policies.
```

So we need to decide what's the best value from the security point of view and also from the usability point of view that will not break things.

 `027` looks good to me as it will provide access to groups as well.

So do I need to get this reviewed or approve? OR I can go ahead and change it to `027`?

Reply:
Let's run this by @cprice-delphix for confirmation, but my preference would be to ship with a more restrictive/secure default umask (`027`). This will also potentially address a number of other CIS findings around permissive file permissions (e.g. in `/var/log` and other places) for files that happened to be created using the default umask.

Reply:
The CIS benchmark PDF defines the `umask` test as:
> 5.5.4 Ensure default user umask is 027 or more restrictive (Automated)

So I'd expect this to address the finding.

Reply:
Thanks for the confirmation, I will make that change.